samples/spring-valueentity-counter/src/main/java/com/example/action/Confirmed.java

```diff
@@ -0,0 +1,16 @@
+package com.example.action;
+
+public class Confirmed {
+
+    private static Confirmed instance = null;
```

Comment:
```suggestion
    final public static Confirmed instance = new Confirmed();
```

And then we can remove the more verbose `getDefaultInstance`

Reply:
I see that we have the same `Confirmed` type in other samples. We could do the same there as well. I mean, have a `final public static instance`.

Btw, it seems that the other variants are using tab instead of space for indentation.

Reply:
Done